wire-ios-request-strategy/Sources/E2EIdentity/E2eIEnrollment.swift

```diff
@@ -318,6 +342,18 @@ public final class E2eIEnrollment: E2eIEnrollmentInterface {
         }
     }
 
+    public func getOAuthRefreshToken()  async throws -> String? {
```

Comment:
Do we expect the refresh token to ever be `nil` or empty? If it ever is, should it be considered an error? Could we return a non-optional `String` and throw an error if the token is nil or empty?